### ISSUE TITLE
Add eval script

### ISSUE THREAD

**Issue by aaprasad** [talmolab/dreem#16]
We're missing an eval script which could be useful. It's pretty easy to write:

```python 
import hydra
import pytorch_lightning as pl
import pandas as pd

from biogtr.config import Config
from models import GTRRunner
@hydra.main(config_path="configs", config_name=None, version_base=None)
def eval(cfg):
      test_cfg = Config(cfg)
      
      dataset = test_cfg.get_dataset(mode="test")
      dataloader = test_cfg.get_dataloader(dataset, mode="test")
      
      ckpt = test_cfg.get_ckpt_path()
      
      model = GTRRunner.load_from_checkpoint(ckpt)
      
      trainer = cfg.get_trainer()
      
      results = trainer.test(model, dataloader)
      
      with open(cfg.out_path, 'w') as f:
        for key, val in results:
               print(f'{key}:{val}')
               f.write(f'{key}:{val}')
 if __name__ == "__main__":
        eval()              
```

**Comment by aaprasad** [comment]
This is low priority, will update when about to hand off to newcomer

**Comment by aaprasad** [comment]
addressed in #67
